foundation_cms/static/js/redesign_main.js

```diff
@@ -1,8 +1,13 @@
 import $ from "jquery";
 import "foundation-sites";
+import { initAllAnimatedImpactNumbers } from "./blocks/impact_numbers_block.js";
 
 $(document).foundation();
 
 console.log(
   "😃 If you see this message, the JS watch and build steps are working properly! 😃",
 );
+
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Hi @mmmavis! I was wondering where the best place to be to init() watchers for blocks that are usable across all pages? For example, I saw that we are doing so in main.js on the old site, however, I wasnt sure if we wanted to take a new approach for this site. 

Thanks!

Reply:
Hey @danielfmiranda , thanks for checking

This is what we have currently 

```bash
foundation_cms/static/js
├── components
│   └── home_page
│       └── kinetic_type_brand_line.js
├── pages
│   └── home_page.js
└── redesign_main.js
```

I'm thinking we can do

```bash
foundation_cms/static/js
├── components
│   └── home_page
│       └── kinetic_type_brand_line.js
│   └── common (<-- new)
│       └──component_1.js
│       └──component_2.js
├── pages
│   └── home_page.js
└── redesign_main.js (<--- and initialize the common components here)
```

Curious what you think about this.